keyboards/lostdotfish/rp2040_orbweaver/keyboard.json

```
@@ -0,0 +1,118 @@
+{
+    "manufacturer": "Lostdotfish",
+    "keyboard_name": "rp2040_orbweaver",
+    "maintainer": "Lostdotfish",
+    "url": "https://geekhack.org/index.php?topic=124092.0",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "rgb_matrix": {
+        "driver": "is31fl3731",
+        "led_flush_limit": 26,
+        "led_process_limit": 5,
+        "max_brightness": 220,
+        "sleep": true,
+        "center_point": [40, 30],
```

All of the defaults except for value are the core's defaults, so no need to specify them. 
```suggestion
        "center_point": [40, 30],
        "default": {
            "val": 127
        },
        "timeout": 1000000
```